conf/cassandra.yaml

```diff
@@ -203,6 +203,12 @@ partitioner: org.apache.cassandra.dht.Murmur3Partitioner
 # data_file_directories:
 #     - /var/lib/cassandra/data
 
+# Directory were Cassandra should store the data of the local system keyspaces.
+# By default Cassandra will store the data of the local system keyspaces in the first of the data directories.
+# This approach ensure that if one of the other disk is lost Cassandra can continue to operate. For extra security
+# this setting allow to store those data on a different directory that provide redundancy.
+# system_data_file_directory: 
```

Comment:
We should add this to [the documentation for `cassandra.yaml`](https://github.com/apache/cassandra/blob/trunk/doc/source/configuration/cass_yaml_file.rst).

Reply:
Done